native-plasma/app/build.gradle

```diff
@@ -19,7 +19,7 @@ model {
 
     android.ndk {
         moduleName = "native-plasma"
-        CFlags.add("-I${file("src/main/jni/native_app_glue")}".toString())
+        cppFlags.add("-I${file("src/main/jni/native_app_glue")}".toString())
```

Comment:
I'd rather not until the issue with instant run is fixed :/ But if you insist I'll :)